HousingRepairsSchedulingApi/Program.cs

```diff
@@ -20,6 +20,12 @@ public static void Main(string[] args)
             Host.CreateDefaultBuilder(args)
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
+                    webBuilder.UseSentry(o =>
+                    {
+                        o.Dsn = Environment.GetEnvironmentVariable("SENTRY_DNS");
+                        o.Debug = true;
```

Comment:
It would be best to only utilise debugging and gathering all trace data only during development.

See these [changes](https://github.com/City-of-Lincoln-Council/HousingManagementSystemApi/pull/8/commits/fa1326e4f9600c893f1c272609aab58d01ad37e3#diff-7ea396d9d66ad1bbfa50d186251c7d299541ce87e64a4a29125287d49c2eff04R27) for the relevant changes.